images/runtime/build-gops.sh

```diff
@@ -25,6 +25,3 @@ for arch in amd64 arm64 ; do
   mkdir -p "/out/linux/${arch}/bin"
   GOARCH="${arch}" CGO_ENABLED=0 go build -ldflags "-s -w" -o "/out/linux/${arch}/bin/gops" github.com/google/gops
 done
-
-x86_64-linux-gnu-strip /out/linux/amd64/bin/gops
-aarch64-linux-gnu-strip /out/linux/arm64/bin/gops
```

Comment:
Manually striping the binaries is not necessary as they are already built with `-ldflags "-s -w"`:
* `-s Omit the symbol table and debug information.`
* `-w Omit the DWARF symbol table.`

[ref](https://pkg.go.dev/cmd/link)